### ISSUE TITLE
broken links in docs

### ISSUE THREAD

**Issue by Laubeee** [chbrandt/dachs-doc#9]
I encountered a couple of broken links... didn't keep track but I thought I'd report it anyway.
Latest example. https://dachs-doc.readthedocs.io/tutorial.html#more-on-importing-recipes the rr/q link is broken.

Can you sum up what is the idea there? I suppose I define a table "imported" in the RD? Does it get automatically filled?

**Comment by msdemlei** [comment]
On Fri, Sep 11, 2020 at 05:08:48AM -0700, Silvan Laube wrote:
> I encountered a couple of broken links... didn't keep track but I thought I'd report it anyway.
> Latest example. https://dachs-doc.readthedocs.io/tutorial.html#more-on-importing-recipes the rr/q link is broken.

Uh -- this is broken in the github CI.  Dang.  We'll have to see how
to fix it.

Meanwhile, could I point you to
https://docs.g-vo.org/DaCHS/tutorial.html#more-on-importing-recipes?
All links should work there.

Carlos: Do you have time to have a quick look on why our custom
interpreted text roles don't work?  Have readthedocs changed their
URL schemas?

> Can you sum up what is the idea there? I suppose I define a table
> "imported" in the RD? Does it get automatically filled?

So, first off: this is only relevant if you do incremental imports of
files that are not themselves published; in the common case that you
just get additional spectra or images now and then, just use

```
  <ignoreSources fromdb="select.accref from \schema.mytable"/>
```

or so.

But if that's not what you have, yes, you'll have to manually keep
track of which files are imported.  And no, DaCHS does not fill any
table saying this for you.  Depending on what the situation is, there
are various ways to do this.  

RR, for instance, has a dispatching grammar that, when it has yielded
all records from a source, yields the row for a table imported, and
there's a make for this.  I'd say that's higher DaCHS-fu.

Unless there are more than a few dozen items per imported file, I'd
probably just store the source path in a column, say, parsed_from:

```
  <map key="parsed_from">\inputRelativePath</map>
```

Have an index over parsed_from, and then do, in your data:

```
  <ignoreSources fromdb="select distinct parsed_from from
    \schema.mytable"/>
```

Would that work for you?  If not, let's take this to dachs-users...

**Comment by msdemlei** [comment]
The original issue, the broken link to the rr/q sample RD, ought to be fixed by commit 44543e1].
